libs/libevhtp/Makefile

```diff
@@ -11,25 +11,28 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libevhtp
-PKG_VERSION:=1.1.6
-PKG_RELEASE:=3
-PKG_LICENSE:=BSD-3-Clause
+PKG_VERSION:=1.2.0
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/criticalstack/libevhtp/tar.gz/$(PKG_VERSION)?
-PKG_HASH:=725dd0a32237b2a097cdc2003d09278a082ccce310aaba7441a5fd8fa8f26635
+PKG_HASH:=d8d98072693f5f68ccd74d327dedfa2f6add4446ac2799689c2f58dd480aa301
+
+PKG_MAINTAINER:=
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
 
-PKG_INSTALL:=1
+CMAKE_INSTALL:=1
```

Comment:
Why is this added?

Reply:
Allows removal of the InstallDev section.

Reply:
I personally think that should be kept

Reply:
git grep CMAKE_INSTALL\: | wc -l
33
git grep cmake.mk | wc -l
68

No clear winner. I prefer CMAKE_INSTALL as it simplifies the Makefile.

Reply:
I agree with @neheb here.

Reply:
Unless I'm reading the code wrong it'll pull in everything (running a regular install) which will lead to possible build bloat(?) and imho obfuscation.

Reply:
It installs everything from ipkg-install to the staging_dir, yes. But disk space is not really a concern for the host. Only for the device running the library. But Build/Install is not touched.

Reply:
They install exactly the same files:
```
$ ls -R ipkg-install/
ipkg-install/:
usr

ipkg-install/usr:
include  lib

ipkg-install/usr/include:
evhtp.h  evthr.h  htparse.h

ipkg-install/usr/lib:
libevhtp.a
```

Reply:
@cotequeiroz 
I was referring in general terms, I doubt it'll be true for all.